app/assets/stylesheets/components/_super-navigation-header.scss

```diff
@@ -0,0 +1,90 @@
+$breakpoint: 769px;
+
+// Remove bottom borders
+.govuk-header__super-navigation,
+.govuk-header__super-navigation-container {
+  border-bottom: 0;
+  margin-bottom: 0;
+  padding-top: 0;
+}
+
+// logo
+.govuk-header__super-navigation-header-logo {
+  padding-top: govuk-spacing(2) + 4;
+  padding-bottom: govuk-spacing(2);
+  margin-bottom: 0;
+}
+
+// header content
+.govuk-header__super-navigation-content {
+  @include govuk-media-query($from: $breakpoint) {
+    text-align: right;
+  }
+}
+
+.govuk-header__super-navigation-items {
+  margin-bottom: 0;
+}
+
+.govuk-header__super-navigation-item {
+  display: flex;
+  margin-right: 0;
+  padding: 0;
+
+  @include govuk-media-query($from: $breakpoint) {
+    display: inline-flex;
+  }
+}
+
+.govuk-header__super-navigation-item-link {
+  padding: govuk-spacing(3) + 2;
+  width: 100%;
+
+  @include govuk-media-query($from: $breakpoint) {
+    width: auto;
+  }
+}
```

Comment:
The new accordion component from the gem had some feedback from the Design System team that the focus state was overly large:

> The focus state taking up the entire button is larger than necessary and is likely to be distracting for users

https://github.com/alphagov/govuk-design-system-backlog/issues/1#issuecomment-839802842

I wonder if the same thing could apply for the focus state and the clickable area for this menu when seen on small and medium screen sizes?

![image](https://user-images.githubusercontent.com/1732331/120806765-26d46f80-c53f-11eb-9261-b8a60178703f.png)

Reply:
It's part of the mobile styling, the clickable area is much larger